lib/src/server/server.dart

```diff
@@ -85,7 +85,10 @@ class Server {
   Service lookupService(String service) => _services[service];
 
   Future<void> serve(
-      {dynamic address, int port, ServerTlsCredentials security}) async {
+      {dynamic address,
+      int port,
+      ServerTlsCredentials security,
+      void Function(String message) logError = print}) async {
```

Comment:
We could also use package logger. Hesitant to include another dependency...

Reply:
Actually, it would probably be better to:
A) Use `package:logging`, or,
B) Don't print anything.

These mostly look like debug statements anyways.

Reply:
After conferring with @lrhn I chose to swallow non-Errors and pass errors on to the zone's error handler